src/main/java/io/airlift/airline/GlobalSuggester.java

```diff
@@ -20,9 +17,9 @@ public class GlobalSuggester
     public Iterable<String> suggest()
     {
         return concat(
-                transform(metadata.getCommandGroups(), CommandGroupMetadata::getName),
-                transform(metadata.getDefaultGroupCommands(), CommandMetadata::getName),
-                concat(transform(metadata.getOptions(), OptionMetadata::getOptions))
-        );
+                concat(
+                        metadata.getCommandGroups().stream().map(CommandGroupMetadata::getName),
+                        metadata.getDefaultGroupCommands().stream().map(CommandMetadata::getName)),
+                metadata.getOptions().stream().flatMap(option -> option.getOptions().stream()))::iterator;
```

Comment:
While I like the approach of using `::iterator`, this actually has the potential to cause issues at runtime

Streams are consumable, and so if you call iterator twice on a stream, it may produce an error. Try running the following:

```java
@Test
    public void testt() {
        Iterable<Integer> it = ImmutableList.of(1, 2, 3, 4).stream()::iterator;
        for(int i = 0; i < 10; i++) {
            System.out.println(Joiner.on(", ").join(it));
        }
    }
```

You could instead use a supplier and it will work

```java
@Test
    public void testt() {
        Iterable<Integer> it = () -> ImmutableList.of(1, 2, 3, 4).stream().iterator();
        for(int i = 0; i < 10; i++) {
            System.out.println(Joiner.on(", ").join(it));
        }
    }
```